app/containers/Forms/FormFieldTemplate.tsx

```diff
@@ -44,8 +44,8 @@ const FormFieldTemplate: React.FunctionComponent<FieldTemplateProps> = ({
       <Col xs={12} className={classNames}>
         <Form.Row>
           {description}
-          {errors}
-          {helpText}
+          <div className="col-12">{errors}</div>
+          <div className="col-12">{helpText}</div>
```

Comment:
Wrapping the errors and helptext (which are shown at the the top of the form) in a full-width container prevents a layout bug where the appearance of errors on validation suddenly squishes the first form field to the side. This way there's a full-width spot preserved for errors even before there are any errors to show.